windows_impl.go

```diff
@@ -0,0 +1,1670 @@
+// Copyright 2017 The Virtual Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+//
+// THIS FILE IS AUTOGENERATED BY generator.go
```

Comment:
Oh I thought it was about syntax (no block comments, build tags separate)
and entirely missed that it could be the wording, sorry. will fix.